bears/general/FilenameBear.py

```diff
@@ -38,6 +46,21 @@ def run(self, filename, file,
         """
         head, tail = os.path.split(filename)
         filename_without_extension, extension = os.path.splitext(tail)
+
+        if file_naming_convention is None:
+            self.warn('Please specify a file naming convention explicitly. '
+                      'Using the default "snake" naming convention...')
+            file_naming_convention = 'snake'
+
+        if file_naming_convention.lower() == 'auto':
+            if extension not in self._language_naming_convention.keys():
```

Comment:
omit `.keys()`, as this is the same:
`extension not in self._language_naming_convention`